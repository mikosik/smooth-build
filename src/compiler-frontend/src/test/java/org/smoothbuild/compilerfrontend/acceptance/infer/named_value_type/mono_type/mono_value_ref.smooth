String stringValue = "abc";
myValue = stringValue;
