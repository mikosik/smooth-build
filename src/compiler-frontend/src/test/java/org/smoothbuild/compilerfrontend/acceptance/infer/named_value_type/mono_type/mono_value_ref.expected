String stringValue<>
  = "abc";
String myValue<>
  = stringValue<>;
